src/commands/mod.rs

```diff
@@ -314,7 +304,14 @@ pub fn main() {
                     crate::utils::crashreporting::try_report_to_sentry(&err);
                 }
                 1
-            }
+            };
+
+            // if the user hit an error, it might be time to run the update
+            // nagger because maybe they tried to do something only newer
+            // versions support.
+            run_sentrycli_update_nagger();
```

Comment:
This now runs unconditionally, which I think makes sense in general. The comment no longer matches, though.

Reply:
It's not true. It'll only run if `run` returns the error, which is the exact same behavior as before, just moved to after original error printing.

Reply:
Oh right, sorry, I read this wrong. All good!